x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/types.ts

```diff
@@ -22,6 +22,19 @@ export interface Engine {
   };
 }
 
+interface SearchRelevanceSuggestions {
+  count: number;
+  curation: {
```

Comment:
Do we have a type for curation already that we could use here?

Reply:
So this is more like a summary/details type than the `CurationSuggestion` type use get back from our endpoints.  These are aggregated details coming from an internal endpoint. We do this for meta engines and crawler as well to track similar top-level engine level information we want to be aware of in the whole Engine UX.  I tried to clarify that with new names in e0ed6a42